nucleus/admin/template/src/main/resources/config/logging.properties

```diff
@@ -140,6 +140,7 @@ jakarta.enterprise.system.core.security.level=INFO
 jakarta.enterprise.system.core.security.web.level=INFO
 jakarta.enterprise.system.core.selfmanagement.level=INFO
 jakarta.enterprise.system.core.transaction.level=INFO
+jakarta.enterprise.system.core.jsch.level=SEVERE
```

Comment:
Ordering: alphabet ;-)

Reply:
I renamed the logger to `jakarta.enterprise.system.core.ssh` and fixed the ordering ;-)